src/app/app-designs/app-designs.component.ts

```diff
@@ -136,12 +138,13 @@ export class AppDesignsComponent implements OnInit, OnDestroy {
         }
       } else {
         this.errorObservable = this.store.pipe(select((store) => store.loader.loadingError)).subscribe((loadingError) => {
-          if (loadingError) {
-            if (!this.dialog.openDialogs.find((dialog) => dialog.id === 'error') && this.router.url === '/interactiveMap') {
```

Comment:
this line it is not working, I dont understand why because it's suppose that `this.dialog.openDialogs` is an array with all the dialogs opened, but I can not find the error dialog...